### PR TITLE
maintainers: add genofire (me)

### PR DESCRIPTION
**Pull request by genofire**
###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/misc/gotify-desktop/default.nix [hunk 1]
```diff
@@ -21,6 +21,6 @@ rustPlatform.buildRustPackage rec {
     description = "Small Gotify daemon to send messages as desktop notifications";
     homepage = "https://github.com/desbma/gotify-desktop";
     license = licenses.gpl3Plus;
-    maintainers = [ maintainers.bryanasdev000 ];
+    maintainers = [ maintainers.bryanasdev000 maintainers.genofire ];
```
Comment (bobby285271) [inline pkgs/tools/misc/gotify-desktop/default.nix:24]: When there are multiple maintainers, we prefer this way:

```suggestion
    maintainers = with maintainers; [ bryanasdev000 genofire ];
```

**Comment by bobby285271** [comment]
Please squash e728d563553453d55122cdd6ffa54a9324432c00 08e690289e78a4d0c11ce231b39dcf9574258d06 3aca25cb089f4410483ec1d68423664a22ae4c99 into one, then it should be good to go.

File: maintainers/maintainer-list.nix [hunk 2]
```diff
@@ -4187,6 +4187,16 @@
...
+  genofire = {
+      name = "genofire";
+      email = "geno+dev@fireorbit.de";
+      github = "genofire";
+      githubId = 6905586;
+      keys = [{
+        longkeyid = "rsa4096/0xFC83907C125BC2BC";
+        fingerprint = "386E D1BF 848A BB4A 6B4A  3C45 FC83 907C 125B C2BC";
+      }];
+  };
```
Comment (bobby285271) [inline maintainers/maintainer-list.nix:4199]: Ooh the indentation here is not correct, we use 2 spaces.

**Review by bryanasdev000 — APPROVED** [review]
Welcome aboard :D

**Comment by bryanasdev000** [comment]
Ops @genofire I think that rebased went a bit too far :P 

- Make sure to rebase on master
- Add your maintainers info and commit
- Add yourself to gotfy-desktop maintainer and make sure that it can build (`nix-build -A gotify-desktop` on nixpkgs root)
- Push it!

**Comment by genofire** [comment]
sorry something goes wrong with rebase - a old master branch version ...

**Review by bryanasdev000 — APPROVED** [review]
LGTM

**Review by bryanasdev000 — APPROVED** [review]

**Review by 0x4A6F — APPROVED** [review]
Welcome. :heart:

**Pull request merged**